src/map.ts

```diff
@@ -123,13 +131,102 @@ export const useMap = () => { useEffect(() => {
     displayFeatureInfo(pixel, evt.originalEvent.target as Element);
   });
 
+  ////////////////////////
+  // Toggle Polygon Button
+  const toggleOn = (btn: JQuery) => {
+    btn.removeClass('toggle-off');
+    btn.addClass('toggle-on');
+    map.removeInteraction(select);
+    map.addInteraction(draw);
+  };
+
+  const toggleOff = (btn: JQuery) => {
+    btn.removeClass('toggle-on');
+    btn.addClass('toggle-off');
+    map.removeInteraction(draw);
+    map.addInteraction(select);
+  };
+
+  const select = new Select({
+    condition: click,
+  });
+
+  map.addInteraction(select);
+
+  const selectedFeatures = select.getFeatures();
+
+  const showOrHideDownloadButton = function() {
+    const btn : JQuery = $('#map-download-btn');
```

Comment:
Hey Scott! I would avoid adding a JQuery dependency for document queries, as this is now supported without dependencies.

```suggestion
    const btn : Element = document.querySelector('#map-download-btn');
```